systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -944,6 +945,24 @@ void testMirrorMakerTlsScramSha() {
         checkRecordsForConsumer(messagesCount, jobReadMessagesForTarget);
     }
 
+    @Test
+    void testEntityUserOperatorValidityAndRenewal() {
```

Comment:
Why is this test in `KafkaST.java`? It could be in `KafkaClusterTest` where it will be much quicker to run and fail much earlier.

Reply:
Can we test deployments here? AFAICS it tests only kafka part of cluster.

Reply:
True, it's not `KafkaCluster`, and it looks like there's not a test class which lets you mane assertions about the `Deployment`. But the `EntityUserOperatorTest` would let you asset about the container, and that's enough for env vars.